docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -80,6 +85,116 @@ export default function App() {
 }
 ```
 
+## Configuration
+
+### Expo config
+
+To configure `expo-splash-screen`, use the built-in [config plugin](/guides/config-plugins) in the Expo config (**app.json** or **app.config.js**) for [EAS Build](/build/introduction) or with `npx expo run:[android|ios]`. The plugin allows you to configure the following properties that cannot be set at runtime and require building a new app binary to take effect:
```

Comment:
```suggestion
To configure `expo-splash-screen`, use the built-in [config plugin](/guides/config-plugins) in the [Expo config](/workflow/configuration/) for [EAS Build](/build/introduction) or with `npx expo run:[android|ios]`. The plugin allows you to configure the following properties that cannot be set at runtime and require building a new app binary to take effect:
```

I think we can just link to the Expo config page rather than specifying file names here. It's a small thing that we are trying to avoid with new info.